fuzzbucket_client/__main__.py

```diff
@@ -576,6 +762,57 @@ def create(self, known_args, _):
         print(self._format_boxes(raw_response["boxes"]), end="")
         return True
 
+    @_command
+    def update(self, known_args, _):
+        matching_boxes = self._find_boxes(known_args.box_match)
+        if matching_boxes is None:
+            log.error(f"no boxes found matching {known_args.box_match!r}")
+            return False
+        payload = {}
+        if known_args.ttl:
+            if known_args.ttl.total_seconds() < MIN_TTL.total_seconds():
+                log.error(f"ttl={known_args.ttl!r} is below the minimum of {MIN_TTL}")
+                return False
+            if known_args.ttl.total_seconds() > MAX_TTL.total_seconds():
+                log.error(f"ttl={known_args.ttl!r} is above the maximum of {MAX_TTL}")
+                return False
+        if known_args.instance_tags:
+            payload["instance_tags"] = {}
+            for pair in known_args.instance_tags.split(","):
+                if ":" not in pair:
+                    continue
+                key, value = [
+                    urllib.parse.unquote(str(s.strip()))
+                    for s in pair.strip().split(":", maxsplit=1)
+                ]
+                log.debug(f"adding instance tag to request key={key!r} value={value!r}")
+                payload["instance_tags"][key] = value
+        if len(payload) == 0 and known_args.ttl is None:
+            log.error(f"no updates specified for {known_args.box_match!r}")
+            return False
+        for matching_box in matching_boxes:
+            box_payload = payload.copy()
+            box_payload["ttl"] = str(
+                int(
+                    (
+                        datetime.datetime.utcnow().timestamp()
+                        - float(matching_box["created_at"])
+                    )
+                    + known_args.ttl.total_seconds()
```

Comment:
Can `known_args.ttl` be `None` here, for example if updates were specified but no TTL?